### PR TITLE
Improve hover effect

### PR DESCRIPTION
**Pull request by EmmanuelDrouin**
- Improve hover effect on components: 
  - Button
  - Card
  - Side menu (+update design)
  - Theme (link)

### CODE REVIEW

**Comment by antho1404** [comment]
It would be nice to have a changelog for each components in order to track easily the modifications.
Maybe you could start on this PR and we do it on a need basis.

TODO: 
- Add a file `CHANGELOG.md` for each (updated) components with the following content 

```markdown
# Changelog

## [Unreleased](https://github.com/mesg-foundation/js-sdk/releases/tag/%40mesg-components%2FCOMPONENT_NAME%40X.X.X)

#### Breaking Changes
#### Improvements
#### Bug fixes
```

- Populate the changelog for this PR with the breaking change/improvements/bug fixes

File: packages/Button/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+#### Breaking Changes
+#### Improvements
+- [98] (https://github.com/mesg-foundation/mesg-components/pull/98/) Refactoring of the scss.
```
Comment (antho1404) [inline packages/Button/CHANGELOG.md]: Just a tiny formatting issue

This should be:
```
- [#98](https://github.com/mesg-foundation/mesg-components/pull/98/) Refactoring of the scss.
```

`#` in front of the number of the pull request and space to make it a link.
Same for all the other changelogs

Comment (EmmanuelDrouin) [reply packages/Button/CHANGELOG.md]: Fixed

**Comment by EmmanuelDrouin** [comment]
@antho1404 ready for review

**Review by antho1404 — APPROVED** [review]

**Pull request merged**